kit/inputs/src/UISeedPhraseInput/UISeedPhraseInput.tsx

```diff
@@ -108,24 +95,18 @@ export const UISeedPhraseInput = React.forwardRef<UIMaterialTextViewRef, UISeedP
 
         const [hasNonLatinCharacters, setHasNonLatinCharacters] = React.useState(false);
 
-        const previousTextRawRef = React.useRef<string | null>(null);
+        const maxWords = React.useMemo(() => Math.max.apply(null, totalWords), [totalWords]);
 
         const onChangeText = React.useCallback(
             (textRaw: string) => {
-                /** To prevent onChangeText from being called twice with one actual text change */
-                if (previousTextRawRef.current === textRaw) {
-                    return;
-                }
-                previousTextRawRef.current = textRaw;
-
```

Comment:
This hack led to bug when the same invalid text was entered twice. 
I couldn't catch calling `onChangeText` twice on change text.